rskj-core/src/main/java/co/rsk/net/light/LightSyncProcessor.java

```diff
@@ -120,44 +123,77 @@ public void sendStatusMessage(LightPeer lightPeer) {
                 block.getNumber(), lightPeer.getPeerIdShort());
     }
 
-    public void sendBlockHeadersMessage(LightPeer lightPeer, byte[] bestBlock, int max, int skip, boolean reverse) {
-        GetBlockHeadersByHashMessage blockHeaderMessage = new GetBlockHeadersByHashMessage(++lastRequestedId, bestBlock, max, skip, reverse);
+    public void sendBlockHeadersMessage(LightPeer lightPeer, byte[] startBlockHash, int maxAmountOfHeaders, int skip, boolean reverse) {
```

Comment:
the method naming is a little confusing, I know they do the same thing, but with different messages, you should rename it reflecting that